### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by justas-kuizinas** [avocode/avocode#889]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.6.0
Source UID: 97146
```
TypeError: Cannot set property 'selectionGroup' of undefined
  at t.<anonymous> (C:\Users\xorrr\AppData\Local\avocode\app-2.6.0\resources\app.asar\node_modules\@avocode\selection-stageview\lib\main.js:1:1936)
  at t.EventEmitter.emit (C:\Users\xorrr\AppData\Local\avocode\app-2.6.0\resources\app.asar\node_modules\@avocode\core\node_modules\eventemitter2\lib\eventemitter2.js:339:22)
  at t.emitSelected (C:\Users\xorrr\AppData\Local\avocode\app-2.6.0\resources\app.asar\node_modules\@avocode\core\lib\selection.js:1:4934)
  at t.setActiveView (C:\Users\xorrr\AppData\Local\avocode\app-2.6.0\resources\app.asar\node_modules\@avocode\core\lib\avocode.js:1:30484)
  at C:\Users\xorrr\AppData\Local\avocode\app-2.6.0\resources\app.asar\node_modules\@avocode\core\lib\avocode.js:1:9757
  at Immediate._onImmediate (C:\Users\xorrr\AppData\Local\avocode\app-2.6.0\resources\app.asar\node_modules\@avocode\core\lib\avocode.js:1:10469)
  at processImmediate [as _immediateCallback] (timers.js:361:17)

```

**Comment by connor-bean** [comment]
Thanks for the report. Can you describe what you were doing when you received this error?

Thanks!
Connor, Avocode Support

**Comment by justas-kuizinas** [comment]
Well was just using your soft, measuring psd layers.

**Comment by connor-bean** [comment]
Can you try downloading our newest beta to see if you still get this error?

We released 2.8.6 last night.

https://manager.avocode.com/download/app-beta/mac/

https://manager.avocode.com/download/app-beta/windows/

https://manager.avocode.com/download/app-beta/linux-ia32/

https://manager.avocode.com/download/app-beta/linux/

**Comment by connor-bean** [comment]
Hey!

We have released the newest version of the Avocode app, version 2.8, which we expect to fix this problem. Please download it at https://avocode.com/download/. I am going to close this ticket but please reopen it if you’re still experiencing this problem in Version 2.8.

Thanks!
Connor, Avocode Support
